package.json

```diff
@@ -8,16 +8,19 @@
     "test": "test"
   },
   "scripts": {
-    "build": "rm -rf lib/{js,css} && npx babel src/js --source-maps --out-dir lib/js && npm run compile-sass && npm run autoprefix",
-    "compile-sass": "node-sass --source-map lib/css/ --source-map-embed false --include-path src/scss/styles.scss src/scss/styles.scss -o lib/css/",
     "autoprefix": "npx postcss lib/css/styles.css --use autoprefixer -map -d lib/css/",
+    "build": "npm run compile-js && npm run compile-sass && npm run autoprefix && npm run copy-uswds -s",
```

Comment:
We've removed the part where we clear out the `lib/` directory first before adding the compiling files. Is that not something we still need to do? To make sure that files we may have deleted are being deleted?

Reply:
For CI it's nice to be completely sure it's clean but I wasn't sure whether it was really needed when you're building over and over locally. Files removed from the project are relatively rare. I can put it in here as well though if you think we should have it.

Reply:
I think it can't hurt! Might as well include it to be extra sure it's reflective of reality.